content/en/docs/what-is-grpc/core-concepts.md

```diff
@@ -202,7 +202,7 @@ Keys are case insensitive and consists of ASCII letters, digits, and special cha
 `.` and must not be start with `grpc-` (which are reserved for gRPC itself).
 Binary-valued keys end in `-bin` while ASCII-valued keys do not.
 
-Metadata is opaque to gRPC itself - it lets the client provide information
+User-defined metadata is not used by gRPC, which allows the client provide information
```

Comment:
`which allows the client _to_ provide ...`

Reply:
Right, thanks for the careful read @dfawley!